setup.py

```diff
@@ -15,16 +15,8 @@
     packages=setuptools.find_packages(),
     install_requires=[
         'numpy',
-        'matplotlib',
-        'ipywidgets'
-        # 'scipy',
-        # 'ipython',
-        # 'vdom',
-        # 'ipywidgets',
-        # 'matplotlib',
-        # 'spikeextractors',
-        # 'pandas',
-        # 'pillow'
+        'spiketoolkit',
```

Comment:
This is an important choice : does spiketoolit depend on spikewidget or the reverse.
If spikewidget depend on spiketoolkit, then we will have only the functional approach for plotting.
We will use the method approach like SpikeComparison.plot_confusion_matrix() and keep only
spikewidget.plot_confusion_matrix(spikecomparison)

Reply:
I think it's better to have spikewidget depending on spiketoolkit. This is helpful (for example in the plot_features I use st to compute pca and it's very short).

I think it's time to remove the plot from the sorting comparison eheheh

Reply:
I am not ready :(
Wait a bit more.